package.json

```diff
@@ -12,7 +12,7 @@
     "build": "node ./scripts/build.js",
     "link": "node ./scripts/link.js",
     "unlink": "node ./scripts/unlink.js",
-    "build:dev": "npm run clean && npm run link && npm run build -- -ni",
+    "build:dev": "npm i && npm run clean && npm run link && npm run build -- -ni && npm run build:demo",
```

Comment:
We should probably change the gh-action as well as it will install the dependencies twice

Reply:
Yes. Updated.